news/2019-02-03-scorewatch-20-january-to-2-february.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: "Scorewatch: 20 January to 2 February"
+date: 2019-02-03 08:00:00 +0000
+---
+
+Did you miss Scorewatch? We're back again with some insane plays to showcase all around the board. Come check out what the most skilled players in osu! have been up to!
+
+![](/wiki/shared/news/2019-02-03-scorewatch-20-january-to-2-february/banner.jpg)
+
+The Scorewatch project has been inactive for quite a while now, but [FayeurS 5](/users/6177263) and I are making an effort to bring it back. For now, we'll be running on a bi-weekly schedule (instead of the previous weekly): we're still working out the organization of this project and don't have a lot of people contributing. Next time, we will have more information about the future of Scorewatch and how you can contribute, if you would like to =).
+
+That being said, let's check out the scores!
+
+---
+
+<iframe width="100%" height="400" src="https://www.youtube.com/embed/B3QgWfUuMA8?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+[idke](/users/4650315): [AugoEidEs [oyasumi]](/beatmapsets/346104#osu/764082) +HR **A** (2432/2472x, 99.52%) #5  
```

Comment:
made a note to look into it next time. I'm not sure that I will be able to do that even by the next post but I will be working on it if it seems like it would fit

not sure if images composed of mostly text is good for news/wiki though. this is a good case for where a "template" might be used to do specially text formatting regularly